cmd/bandd/cmd/root.go

```diff
@@ -71,12 +83,13 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 }
 
 func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
-	authclient.Codec = encodingConfig.Marshaler
+	cfg := sdk.GetConfig()
+	cfg.Seal()
```

Comment:
Please check where we should seal the config? here or in `SetBech32AddressPrefixesAndBip44CoinType`